### PR TITLE
[DOCS-7856] updating network topology map

### PR DESCRIPTION
**Pull request by aliciascott**
### Merge instructions

- [ ] Please merge after reviewing

### CODE REVIEW

File: content/en/network_monitoring/devices/network_topology_map.md [hunk 1]
```diff
@@ -15,62 +15,209 @@ further_reading:
...
-3. If you're using [device autodiscovery][3], enable `snmp_listener.collect_topology: true` in the `datadog.yaml` file.
+2. Datadog Agent version 7.52 or later is installed.
+
+## Navigation options
+
+In the Network Topology Map, the following navigation options are available:
+
+### View
+
+1. Under **View By**, select to visualize devices in groups according to tags:
```
Comment (michaelcretzman) [inline content/en/network_monitoring/devices/network_topology_map.md:39]:
```suggestion
1. In **View By**, use tags to select how you want to visualize devices: 
```

Comment (michaelcretzman) [reply content/en/network_monitoring/devices/network_topology_map.md:39]: just a suggestion.

File: content/en/network_monitoring/devices/network_topology_map.md [hunk 2]
```diff
@@ -15,62 +15,209 @@ further_reading:
...
+{{< img src="/network_device_monitoring/network_topology_map/device_topology_color_by_device.png" alt="The navigation option, with view by color selected, highlighting view by device state" style="width:80%;" >}}
+
+   The following are the definitions of the nodes for each color state:
+   <div style="width:80%; margin: 0 auto;">
+
+   | Color    | Description               |
+   |----------|---------------------------|
+   | Green   | Device is reachable.      |
+   | Red   | Issue with device, such as unreachable through SNMP.  |
+   | Gray    | The device is monitored by NDM, however, no data has been received. For example, if the ping wasn't configured and you opted to `color by` **Ping State** in the Device Topology Map, the devices are displayed in gray. |
```
Comment (michaelcretzman) [inline content/en/network_monitoring/devices/network_topology_map.md:59]:
```suggestion
   | Gray    | Device is monitored by NDM; however, no data has been received. For example, if the ping wasn't configured and you opted to `color by` **Ping State** in the Device Topology Map, the devices are displayed in gray. |
```

Comment (michaelcretzman) [reply content/en/network_monitoring/devices/network_topology_map.md:59]: you weren't using articles at the beginning of the other Descriptions, so I removed this one.

I both Ping and ping in the doc. You might want to just use ping, whether it's a noun or a verb. Very common usage.

File: content/en/network_monitoring/devices/network_topology_map.md [hunk 3]
```diff
@@ -15,62 +15,209 @@ further_reading:
...
+
+   </div>
+
+### Filter devices
+
+3. Under **Filter Devices**, gain further granular control over what devices are shown on the Device Topology Map. 
+
+{{< img src="/network_device_monitoring/network_topology_map/device_topology_filter_devices_hide.png" alt="The navigation option, with view filer option selected, toggling by hide unmonitored devices " style="width:80%;" >}}
+
+**Note:** The filter devices setting impacts what devices are shown on the Device Topology Map for _all_ queries you may make, such as if you filter by a device facet in the search bar.
```
Comment (michaelcretzman) [inline content/en/network_monitoring/devices/network_topology_map.md:70]:
```suggestion
**Note:** The **Filter Devices** setting impacts what devices are shown on the Device Topology Map for _all_ queries you might make. For example, if you filter by a device facet in the search bar. 
```

File: content/en/network_monitoring/devices/network_topology_map.md [hunk 4]
```diff
@@ -15,62 +15,209 @@ further_reading:
...
+
+**Note:** The filter devices setting impacts what devices are shown on the Device Topology Map for _all_ queries you may make, such as if you filter by a device facet in the search bar. 
+
+Hide _N_ Unmonitored Devices - Turned OFF by default.
+: Toggling this on hides devices on the Device Topology Map that are not directly monitored by Network Device Monitoring, but still discovered by LLDP/CDP, and shown on the map from adjacent devices that are monitored by Network Device Monitoring.
+
+Hide _N_ Unconnected Devices - Turned ON by default.
+: Toggling this on hides any devices that have no link connections. Devices can be unconnected for reasons such as improper configuration, or the device does not support [LLDP/CDP](#troubleshooting).
+
+### Icon legend
```
Comment (michaelcretzman) [inline content/en/network_monitoring/devices/network_topology_map.md:78]: this section is under **Navigation options**. Is it an option they can control? If not, consider moving it.

File: content/en/network_monitoring/devices/network_topology_map.md [hunk 5]
```diff
@@ -15,62 +15,209 @@ further_reading:
...
-
-```yaml
+For CDP data
+```yaml:
 sudo -u dd-agent datadog-agent snmp walk <DEVICE_IP> 1.3.6.1.4.1.9.9.23
 ```
 
+### Missing connections or links 
+
+If your device is exposing topology data with LLDP or CDP but some of the connections are missing, ensure that the "Hide _N_ Unmonitored Devices" toggle is off.
```
Comment (michaelcretzman) [inline content/en/network_monitoring/devices/network_topology_map.md:194]: do you want to use quotes around UI elements? Typically, we bold them.

File: content/en/network_monitoring/devices/network_topology_map.md [hunk 6]
```diff
@@ -15,62 +15,209 @@ further_reading:
...
+The borderless or black nodes on the Device Topology Map can represent devices discovered with LLDP or CDP that are not configured to be monitored with NDM, or devices discovered with LLDP or CDP that can not be resolved to the equivalent [monitored device](#device-resolution).
+
+## Device resolution 
+
+The Device Topology Map provides an overview of the devices monitored with NDM and their physical connections. The topology links data is based on LLDP (Link Layer Discovery Protocol) or CDP (Cisco Discovery Protocol) information collected with SNMP. 
+The connections discovered with LLDP or CDP can correspond to devices already monitored with SNMP. The device resolution consists in matching the discovered device to the monitored device.
+
+### Device resolution failures
+
+The device resolution can fail if the device is not monitored with NDM, or the LLDP or CDP data is insufficient to match the discovered device to the monitored device.
```
Comment (michaelcretzman) [inline content/en/network_monitoring/devices/network_topology_map.md:218]: It seems like these requirements are mentioned multiple times in the topic. Consider just mentioning them in the Prereq's and linking to that. Just easier than repeating it, IMO.

**Review by michaelcretzman — APPROVED** [review]
Approved with the assumption that you'll consider my suggestions. Nice work. Good topic.

File: content/en/network_monitoring/devices/network_topology_map.md (hunk 4 above)
Comment (aliciascott) [reply content/en/network_monitoring/devices/network_topology_map.md:77]: Yes it's a toggle, thanks for the tip though!

File: content/en/network_monitoring/devices/network_topology_map.md (hunk 5 above)
Comment (aliciascott) [reply content/en/network_monitoring/devices/network_topology_map.md:193]: let me ask the PM about this, thanks!

**Pull request merged**